### PR TITLE
Video detail page

### PR DESCRIPTION
**Pull request by MelizzaP**

### CODE REVIEW

**Comment by MelizzaP** [comment]
test link: http://video-series-page.test.theonion.com/series/tech-trends

**Comment by MelizzaP** [comment]
@spra85 @jmelvnsn @collin ready for some eyes

File: bulbs/videos/templates/videos/series-page.html [hunk 1]
```diff
@@ -15,26 +15,65 @@
...
+    <div class="ad-alert">Advertisement</div>
+    {% dfp_ad "non-mobile-leaderboard" %}
+    {% dfp_ad "mobile-only-cube" %}
+  </div>
+{% endif %}
 
 <section class="video-grid">
   <header>
     <h2>Episodes</h2>
+    {% if channel.slug != 'the-av-club' %}
```
Comment (spra85) [inline bulbs/videos/templates/videos/series-page.html]: I feel like one of these should be the default we decide on for the django-bulbs version, and then either AVC or Onion extends and overrides a block in the template.

**Comment by mparent61** [comment]
yay tests

File: bulbs/videos/templates/videos/series-page.html [hunk 2]
```diff
@@ -15,26 +15,65 @@
  <p>Watch The Latest Episode</p>
 </section>
 
-<figure class="video-container">
```
Comment (jmelvnsn) [inline bulbs/videos/templates/videos/series-page.html]: Did you check these edits out on AVC? Just want to make sure they don't break any existing views

**Comment by jmelvnsn** [comment]
Looks good. Just confirm that these changes to existing templates don't break any AVC views

**Pull request merged**